token/cli/src/main.rs

```diff
@@ -4293,6 +4366,22 @@ async fn process_command<'a>(
             )
             .await
         }
+        (CommandName::WithdrawExcessLamports, arg_matches) => {
+            let (signer, authority) =
+                config.signer_or_default(arg_matches, "owner", &mut wallet_manager);
+            push_signer_with_dedup(signer, &mut bulk_signers);
```

Comment:
You'll need to only do this if it's *not* a multisig, ie
```suggestion
        if config.multisigner_pubkeys.is_empty() {
            push_signer_with_dedup(signer, &mut bulk_signers);
        }
```
Otherwise, it'll try to sign with the multisig address